packages/webdriverio/package.json

```diff
@@ -59,5 +59,8 @@
     "wdio-config": "^5.0.0-alpha.9",
     "wdio-logger": "^5.0.0-alpha.9",
     "webdriver": "^5.0.0-alpha.9"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0"
```

Comment:
@christian-bromann I couldn't get `lerna build` to run successfully without `babel-cli` installed either in the project or globally on my local. I'll carry on with my global install.